#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_PNG_imagemap.dpatch by Eugen Dedu <Eugen.Dedu@pu-pm.univ-fcomte.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use PNG instead of GIF in HTML image maps.

@DPATCH@
diff -urNad transfig~/doc/fig2dev.1 transfig/doc/fig2dev.1
--- transfig~/doc/fig2dev.1
+++ transfig/doc/fig2dev.1
@@ -473,7 +473,7 @@
 that have href="some_html_reference" in their comments.
 Any Fig object except compound objects may used for this.
 Usually, besides generating the map file, you would also
-generate a GIF file, which is the image to which the map refers.
+generate a PNG file, which is the image to which the map refers.
 .sp
 For example, you may have an xfig drawing with an
 imported image that has the comment
diff -urNad transfig~/fig2dev/dev/genmap.c transfig/fig2dev/dev/genmap.c
--- transfig~/fig2dev/dev/genmap.c
+++ transfig/fig2dev/dev/genmap.c
@@ -172,7 +172,7 @@
   fprintf(tfp, "<P>You can copy the following lines into your HTML document.\n");
   fprintf(tfp, "You may need to edit the name of the image file in the first line.</P>\n");
   fprintf(tfp, "\n");
-  fprintf(tfp, "<IMG SRC=\"%s.gif\" USEMAP=\"#%s\">\n", basename, basename);
+  fprintf(tfp, "<IMG SRC=\"%s.png\" USEMAP=\"#%s\">\n", basename, basename);
   fprintf(tfp, "<MAP NAME=\"%s\">\n", basename);
 
   ref = is_link(objects->comments);
